driver_cpl/driver/cesm_comp_mod.F90

```diff
@@ -1056,6 +1077,9 @@ subroutine cesm_init()
    !  components will set them to true for the purposes of symmetry
    !-----------------------------------------------------------------------------
 
+   call t_startf('cesm_init')
```

Comment:
Timers now include the prefix related to they're model, please add back the 'CPL:' string to timers in the coupler